moola/adaptors/dolfin_vector.py

```diff
@@ -43,7 +43,6 @@ def __init__(self, V, inner_product="L2", map_operator=None, inverse = "default"
         self.map_operator = map_operator
         if inverse in ("default", "lu"):
             self.map_solver = dolfin.LUSolver(self.map_operator)
-            self.map_solver.parameters["reuse_factorization"] = True
```

Comment:
Please implement the new way of setting this flag instead of removing it

Reply:
There is no way of setting this flag. It is completly removed and reises factorization if it can

Reply:
I see. Than this is the best we can do I guess.